polygeist/tools/cgeist/Test/Verification/sycl/functions.cpp

```diff
@@ -101,6 +101,36 @@ SYCL_EXTERNAL void range_size(sycl::range<2> r) {
   keep(r.size());
 }
 
+// CHECK-MLIR-LABEL: func.func @_Z25nd_range_get_global_rangeN4sycl3_V18nd_rangeILi2EEE(%{{.*}}: !sycl_nd_range_2_) attributes {llvm.cconv = #llvm.cconv<spir_funccc>, llvm.linkage = #llvm.linkage<external>, passthrough = ["norecurse", "nounwind", "convergent", "mustprogress"]} {
```

Comment:
Couple of nits: 
  - would be nice to split these lines and add the expected attributes CHECK using CHECK-SAME
  - given that we expect all of these functions to have the same attributes would be nice to store the first match for attributes in a variable and reuse the defined variable in subsequent CHECKs 
  
 Perhaps we can improve the tests in a separate 'NFC' PR